app/services/prophet_ratings/team_season_stats_aggregator.rb

```
@@ -25,9 +25,15 @@ def initialize(season: Season.current, as_of: Time.current)
       @as_of = as_of
     end
 
+    ##
+    # Aggregates and updates statistical metrics for all team seasons in the specified season up to the cutoff date.
+    #
+    # For each team season with finalized games before the cutoff, computes average stats, efficiency standard deviations, volatility metrics, home court advantages, and win/loss counts, then updates the corresponding record with the results.
```

RuboCop (1.76.1)

[convention] Line is too long. [242/140]

(Layout/LineLength)